src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/IllegalTypeCheck.java

```diff
@@ -447,9 +447,7 @@ public void beginTree(DetailAST rootAST) {
         illegalShortClassNames.clear();
 
         for (String s : illegalClassNames) {
-            if (s.indexOf('.') == -1) {
```

Comment:
This is one of the more controversial changes IMO, but I agree it will not have an impact and I am seeing no way to kill the mutation. This if statement prevents fully qualified classes being put into `illegalShortClassNames` but we strictly only match short class names to this map, so there is no way to kill this mutation so we can't check against what won't happen.